src/hooks/validate-animal.js

```diff
@@ -0,0 +1,13 @@
+
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
I wonder if this is the standard naming 😅 To me, vallidate sounded like you're checking the schema on a create or update action, but I don't how I would call this. 

`verify-animal`? `verify-animal-exists`?  `find-animal`?

Reply:
whatever you like it is ok.